OpenWeather/App/Search/SearchLocationScreen.swift

```diff
@@ -18,19 +18,19 @@ import OpenBytesNavigation
 import SwiftUI
 
 struct SearchLocationScreen: View {
-    @ObservedObject var viewModel: SearchLocationViewModel
-
-    init(viewModel: SearchLocationViewModel) {
-        self.viewModel = viewModel
-    }
+    @StateObject var viewModel: SearchLocationViewModel
 
     var body: some View {
         viewModel.view { content in
             List {
                 ForEach(content.result) { location in
-                    LocationResultItem(name: location.name)
+                    // TODO: @0xLeif, a bit confused, is there a memory leak as I expected?
```

Comment:
Hmmm let me load this locally and check to see. Not sure with just looking at it here.

Reply:
I don't think there is a memory leak here.. I think it is in the `FavoritesViewModel` https://github.com/0xOpenBytes/OpenWeather/pull/22/files/7ef14122e1af127f5af5d88812a98c62f269edf1#r1195757611